config/database.go

```diff
@@ -293,7 +293,13 @@ func (ds *DatabaseStore) RemoveFile(name string) error {
 
 // String returns the path to the database backing the config, masking the password.
 func (ds *DatabaseStore) String() string {
-	u, _ := url.Parse(ds.originalDsn)
+	tcpStripper := regexp.MustCompile(`@tcp\((.*)\)`)
+
+	// Remove @tcp and the branches from the host and parse the rest as URl
```

Comment:
Should this read `parentheses`, not `branches`?
```suggestion
	// Remove @tcp and the parentheses from the host and parse the rest as a URL
```

Reply:
Sure. Messed up the words there.